plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -0,0 +1,1142 @@
+                    - See I(display_name)
+            displayNameHtml:
+                description:
+                    - See I(display_name_html)
+            failureFactor:
+                description:
+                    - See I(failure_factor)
+            maxDeltaTimeSeconds:
+                description:
+                    - See I(max_delta_time_seconds)
+            maxFailureWaitSeconds:
+                description:
+                    - See I(max_failure_wait_seconds)
+            minimumQuickLoginWaitSeconds:
+                description:
+                    - See I(minimum_quick_login_wait_seconds)
+            quickLoginCheckMilliSeconds:
+                description:
+                    - See I(quick_login_check_milli_seconds)
+            waitIncrementSeconds:
+                description:
+                    - See I(wait_increment_seconds)
+
+    browser_flow:
+        type: str
+        description:
+            - Authentication flow binding for browser flow.
+        aliases:
+            - browserFlow
+
+    browser_security_headers:
+        type: dict
+        description:
+            - Browser security headers for this realm.
+        aliases:
+            - browserSecurityHeaders
+        suboptions:
+            contentSecurityPolicy:
+                type: str
+                description:
+                    - Content-Security-Policy (CSP) browser header for this realm.
+            xContentTypeOptions:
+                type: str
+                description:
+                    - X-Content-Type-Options browser header for this realm.
+            xFrameOptions:
+                type: str
+                description:
+                    - X-Frame-Options browser header for this realm.
+            xRobotsTag:
+                type: str
+                description:
+                    - X-Robots-Tag browser header for this realm (see U(https://developers.google.com/search/reference/robots_meta_tag))
+                    - C(none) for no header.
+            xXSSProtection:
+                type: str
+                description:
+                    - X-XSS-Protection browser header for this realm.
+            strictTransportSecurity:
+                type: str
+                description:
+                    - Strict-Transport-Security (HSTS) browser header for this realm.
+
+    brute_force_protected:
+        description:
+            - Is brute force detection enabled for this realm or not.
+        aliases:
+            - bruteForceProtected
+        type: bool
+
+    client_authentication_flow:
+        type: str
+        description:
+            - Authentication flow binding for client authentication flow.
+        aliases:
+            - clientAuthenticationFlow
+
+    default_groups:
+        type: list
+        elements: str
+        description:
+            - List of default groups for this realm. Usually with a prepended path, i.e. C(/test) for group C(test).
```

Comment:
```suggestion
            - List of default groups for this realm. Usually with a prepended path, in other words, C(/test) for group C(test).
```
the Latin phrase. Or maybe `for example` is implied?